tests/units/online/test_mobile_base.py

```diff
@@ -73,6 +74,9 @@ def test_reset_odometry(reachy_sdk_zeroed: ReachySDK) -> None:
         assert not np.isclose(odom["x"], 0.0, atol=0.3)
         assert not np.isclose(odom["y"], 0.0, atol=0.2)
         assert not np.isclose(odom["theta"], 0.0, atol=20)
```

Comment:
There is a lot of variance in the expected atols. I understand these where handtuned back when the tests were run in Gazebo. Now that these will be run in fake mode, the values should be very precise. Can you set them all at the same values? My rule of thumb is that 1mm == 1°